AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 stripe.com##.Notification[data-js-id="CookieSettingsNotification"]
+sberdisk.ru##.modal
```

Comment:
Too wide rule. Better change it to `sberdisk.ru##.wrapper > footer + div.modal`.